lab_3_ann_retriever/main.py

```diff
@@ -24,6 +23,7 @@ def save(self) -> dict:
             dict: State of the Node instance
         """
 
+
```

Comment:
revert

Reply:
Done

Reply:
again - I see this changes, so you are not done with it